skbio/sequence/_rna.py

```diff
@@ -135,6 +135,73 @@ def degenerate_map(cls):
     def _motifs(self):
         return _motifs
 
+    @stable(as_of="0.4.0-dev")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        RNA sequence is assumed to be the coding strand. Uracil (U) is
```

Comment:
not neccessarily. There can also be cases of that I have a ncRNA transcript
and want to get its DNA strand. That was why I didn't have "coding strand"
in the docstring previously... but it seems there are no good terminology
for that. Per @jarideout 's suggestion,  I can see it is useful to
distinguish coding strand and template strand. How about this "It returns
the coding DNA strand of the RNA sequence, i.e. Uracil (U) is replaced with
thymine (T) in the reverse transcribed sequence."?

On Mon, Aug 17, 2015 at 2:21 PM, Evan Bolyen <notifications@github.com>
wrote:

> In skbio/sequence/_rna.py
> <https://github.com/biocore/scikit-bio/pull/1057#discussion_r37238466>:
>
> > @@ -135,6 +135,73 @@ def degenerate_map(cls):
> >      def _motifs(self):
> >          return _motifs
> >
> > +    @stable(as_of="0.4.0-dev")
> > +    def reverse_transcribe(self):
> > +        """Reverse transcribe RNA into DNA.
> > +
> > +        RNA sequence is assumed to be the coding strand. Uracil (U) is
>
> This doesn't make a lot of sense, the returned DNA is the coding strand,
> not really the RNA which is assumed to be mRNA in this context.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1057/files#r37238466>.
>